backend/app/adapter/graphql/graphql_test.go

```diff
@@ -43,6 +43,6 @@ func TestGraphQlAPI(t *testing.T) {
 
 	logger := mdtest.NewLoggerFake(mdtest.FakeLoggerArgs{})
 	tracer := mdtest.NewTracerFake()
-	graphqlAPI := NewShort(&logger, &tracer, retriever, creator, verifier, authenticator)
+	graphqlAPI := NewShort(&logger, &tracer, retriever, creator, nil, verifier, authenticator)
```

Comment:
Let's initialize `changelog.ChangeLog` with Fake or dummy dependencies instead of passing `nil`. This would cause crashes if we change the test to actually call GraphQL APIs in the future.